### PR TITLE
Add checks for openj9 test prerequsites

### PR DESCRIPTION
**Pull request by nikolamilijevic1**
- added code to ouptut the version of openj9 prerequisites ant, make, curl and perl in MachineInfo

Signed-off-by: Nikola Milijevic <nmilijev@uwaterloo.ca>

### CODE REVIEW

**Comment by nikolamilijevic1** [comment]
@llxia @renfeiw

File: src/org/openj9/envInfo/MachineInfo.java [hunk 1]
```diff
@@ -22,6 +22,9 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.lang.management.ManagementFactory;
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```
Comment (renfeiw) [inline src/org/openj9/envInfo/MachineInfo.java:27]: You can remove these import for this PR.

Comment (nikolamilijevic1) [reply src/org/openj9/envInfo/MachineInfo.java]: removed👍🏼

**Comment by llxia** [comment]
Could you provide a Grinder link? If it is the internal Grinder, a partial link with the job number is fine. Thanks

**Comment by nikolamilijevic1** [comment]
@llxia https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder_TKG/ #704

**Pull request merged**